### PR TITLE
Feature/rooms display

### PR DESCRIPTION
**Pull request by edignot**
### Type of change made:
- [ ] Bugfix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [x] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Styling- no new features
### Why is this change required? What problem does it solve?
- Fixed date input picker
- Display rooms by date
- Log out buttons
### Were there any challenges that arose while implementing this feature? If so, how were the addressed?
- 
### Images/Gifs
Screen Shot 2020-04-18 at 12 34 24
Screen Shot 2020-04-18 at 12 34 45
### Files modified:
- [x]  index.html
- [x]  user-login.html
- [x]  admin-login.html
- [x]  user.html
- [x]  admin.html
- [x]  css/_admin.scss
- [x]  css/_user.scss
- [x]  css/_footer.scss
- [x]  css/_login.scss
- [x]  css/_user-admin-login.scss
- [x]  css/base.scss
- [ ]  css/_variables.scss
- [x]  css/_mixins.scss
- [x]  css/_media-queries.scss
- [x]  index.js
- [ ]  utils.js
- [ ]  BookingHandler.js
- [ ]  Hotel.js
- [ ]  test/bookingHandler.js
- [ ]  test/hotel-test.js
- [ ]  test/bookings-test-data.js
- [ ]  test/rooms-test-data.js
- [ ]  test/users-test-data.js

### CODE REVIEW

File: src/BookingHandler.js [hunk 1]
```diff
@@ -0,0 +1,35 @@
+class BookingHandler {
```
Comment (kimalajoy) [inline src/BookingHandler.js:1]: I really like how you have the fetch here in this class

File: src/admin.html [hunk 2]
```diff
@@ -11,8 +11,8 @@
 
 <body>
   <header class="admin-header">
-    <div class="title-wrapper">
-      <button id="logout" class="admin-logout">Logout</button>
+    <button id="logout" class="admin-logout">Logout</button>
```
Comment (kimalajoy) [inline src/admin.html:14]: A log out button!? Fancy!!  🤓

**Comment by edignot** [comment]
@hannahhch  @thatpamiam

File: src/BookingHandler.js [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+          'Content-Type': 'application/json'
+        },
+        body: JSON.stringify(body)
+      })
+      .catch(err => {
+        alert(error);
+      });
+  }
+
+  cancelBooking() {
```
Comment (kimalajoy) [inline src/BookingHandler.js:30]: This isn't doing anything right now, but I assume you'll have a method here

Comment (edignot) [reply src/BookingHandler.js:30]: Yes, it's just wireframe

**Comment by kimalajoy** [comment]
Merging! Keep up the great work. Your naming seems really solid and your booking handler class seems well planned! I can let you know that you are able to delete the DS store file

**Pull request merged**